setup.py

```diff
@@ -39,7 +39,7 @@ def read_version():
     "numpy>=1.9.0",
     "protobuf>=3.1",
     "protobuf3-to-dict>=0.1.5",
-    "smdebug_rulesconfig==1.0.0",
+    "smdebug_rulesconfig==1.0.1",
```

Comment:
If we change the requirement to `>=` can we avoid the need to submit a PR every time we update rulesconfig?

Reply:
Understood but the point of pinning a version is so that we submit a PR and verify that the sagemaker SDK tests pass with the new version of rulesconfig. This accomplishes two things:
- If a rulesconfig release occurs and it turns out to be incompatible with the current sagemaker SDK version, then the sagemaker SDK functionality won't be broken immediately (because the version is pinned to an older version of rulesconfig). If sagemaker SDK changes are required with a new rulesconfig release, we mandate that the rulesconfig gets released first, then make a new PR in sagemaker SDK with the changes and bump in rulesconfig version (like this PR).
- A new sagemaker SDK release should happen for each rulesconfig update, regardless of whether SDK changes are required for the rulesconfig update. We don't want users to have any direct interaction with rulesconfig, installing the latest sagemaker SDK should always install the latest rulesconfig. If we use `>=`, user may have to manually reinstall latest rulesconfig even with latest sagemaker SDK.

Reply:
-- For point 1; nothing stops us from reversing this logic if there is a need to update the sdk before releasing rulesconfig.

-- For point 2; How will this scenario occur? Every-time a the sagemaker sdk is built; it will pull the latest rulesconfig ensuring that it is always built with the latest rulesconfig.

Reply:
> -- For point 1; nothing stops us from reversing this logic if there is a need to update the sdk before releasing rulesconfig.

Why would there be a need to update the SDK before releasing rulesconfig?

> -- For point 2; How will this scenario occur? Every-time a the sagemaker sdk is built; it will pull the latest rulesconfig ensuring that it is always built with the latest rulesconfig.

If the user installed sagemaker SDK before a rulesconfig release. Then to have the latest rulesconfig features, they would have to manually install the latest version of rulesconfig, instead of the latest version of sagemaker (which is pinned to latest version of rulesconfig). So, user should have no direct interaction with rulesconfig.